tests/charm++/pup_buffer/Makefile

```diff
@@ -0,0 +1,31 @@
+-include ../../common.mk
+CHARMDIR = ../../..
+CHARMC = $(CHARMDIR)/bin/charmc $(OPTS)
+
+all: pup_buffer
+
+OBJS = pup_buffer.o
+
+pup_buffer: $(OBJS)
+	$(CHARMC) -language charm++ -o pup_buffer $(OBJS) -module CommonLBs
+
+cifiles: pup_buffer.ci
+	$(CHARMC)  pup_buffer.ci
+	touch cifiles
+
+pup_buffer.o: pup_buffer.C cifiles
+	$(CHARMC) -c pup_buffer.C
+
+test: all
+	$(call run, +p1 ./pup_buffer +balancer RotateLB)
+	$(call run, +p4 ./pup_buffer +balancer RotateLB)
+
+smptest: all
+	$(call run, +p2 ./pup_buffer ++ppn 2 +balancer RotateLB)
+	$(call run, +p4 ./pup_buffer ++ppn 2 +balancer RotateLB)
+	$(call run, +p2 ./pup_buffer +noCMAForZC ++ppn 2 +balancer RotateLB)
```

Comment:
if/when Evan's patch for conditionally running these is merged, should use that

Reply:
It's already merged (#2454 )

Reply:
Removed the `+noCMAForZC` run commands because this currently doesn't use ZC internally. Will add back these run statements with the CMA makefile clause, when I internally use ZC.